### PR TITLE
Yukon Childrens arts amount

### LINKED ISSUE #170: Children's arts amount
[Reference ](https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/yukon/5011-pc/information-residents-yukon.html#P4_58326)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #170

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Children's arts amount.
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]:
```suggestion
    - Yukon children's arts amount.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+description: Non-refundable tax credits in Ontario.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml:1]:
```suggestion
description: Non-refundable tax credits in Yukon.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+values:
+  2022-01-01:
+    - yt_childrens_arts_amount
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml:4]: where does it say this is a non-refundable credit?

**Review by MaxGhenis — COMMENTED** [review]
can you find a legal code?

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml (hunk 3 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml:4]: [Source ](https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/yukon/5011-pc/information-residents-yukon.html) - lists the program under "Part B" -> yukon non refundable credits

Comment (MaxGhenis) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml:4]: OK could you add https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/yukon/5011-pc/information-residents-yukon.html#P4_partb as a reference? then git pull upstream master will fix the test failure

**Pull request merged**